onward/app/controllers/TechFeedbackController.scala

```diff
@@ -50,13 +52,44 @@ class TechFeedbackController(ws: WSClient, val controllerComponents: ControllerC
 
   }
 
-  def techFeedback(path: String): Action[AnyContent] = Action { implicit request =>
-    val page = model.SimplePage(MetaData.make(
-      request.path,
-      Some(SectionId.fromId("info")),
-      "Thanks for your report"
-    ))
-    Cached(900)(RevalidatableResult.Ok(views.html.feedback(page, path)))
+  def techFeedback(path: String): Action[AnyContent] = {
+
+    Action.async { implicit request =>
+
+      val page = model.SimplePage(MetaData.make(
+        request.path,
+        Some(SectionId.fromId("info")),
+        "Thanks for your report"
+      ))
+
+      Try(ws.url(Configuration.feedback.feedbackHelpConfig)
```

Comment:
Wrapping a Future with Try won't catch async errors, you'll need to do `Future.recover`

Reply:
It would also be worth logging the error or putting it on the output page